components/rangeSlider/slider.css

```diff
@@ -0,0 +1,135 @@
+.emoji-slider {
+  -webkit-appearance: none;
```

Comment:
This is not needed. It will be generated from browser side. You can remove.

Reply:
Hey I believe having a fallback is always better as the in older version of browsers may fail at supporting some times.